deploy/simulation/sign_multi_adapted.toml

```diff
@@ -11,8 +11,8 @@ Machines, ppm, bf
 16, 4, 7
 16, 8, 8
 16, 16, 10
-16, 32, 10
-16, 64, 10
-16, 128, 10
-16, 256, 10
-16, 512, 10
+16, 32, 12
+16, 64, 13
+16, 128, 14
+16, 256, 15
+16, 512, 16
```

Comment:
why changing the BF factor at the same time? Or why do you do it now and not before?  Just out of curiosity.